collect_app/src/main/java/org/odk/collect/android/utilities/PermissionUtils.java

```diff
@@ -88,13 +90,50 @@ public void onPermissionRationaleShouldBeShown(List<PermissionRequest> permissio
                 .check();
     }
 
+    public static void requestCameraPermission(@NonNull Activity activity, @NonNull PermissionListener action) {
```

Comment:
How about passing a `Context` instead of an `Activity` so you don't need to cast the parameters?

Reply:
I think it's safer to accept the activity since `Dexter.withActivity(activity)` requires that. Putting a context would imply that any context can be passed.

Reply:
Got it, that makes sense!